modal.js

```diff
@@ -29,4 +29,30 @@ modalBtnNextPage.forEach((modalBtnNextPage) => {
       currentStep += 1;
     }
   });
-});
\ No newline at end of file
+});
+
+/* FETCH API BOOKING 1 */
+const dateField = document.querySelector(".modal__date");
+dateField.setAttribute("min", new Date().toISOString().split('T')[0]);
+let timeSlots;
```

Comment:
These (like many of your variables below) are creating global (or module-scoped) variables. I would try to wrap all of this in a function instead.